client/src/pages/Challenge.tsx

```diff
@@ -2,11 +2,18 @@
 
 import { Container, Row, Col, ButtonToolbar, Button, AccordionItem, AccordionHeader, AccordionBody, Accordion } from "react-bootstrap";
 import { useEffect, useState } from "react";
-import { useParams } from "react-router-dom";
+import { useParams, useLocation } from "react-router-dom";
 import { ChallengeDetails } from "../types/challengeDetails";
-// import { getDifficulty } from "../utils";
 import DifficultyBadge from "../components/DifficultyBadge";
 
+function useScrollToTop() {
```

Comment:
Better way to do this is to use a [ScrollRestoration](https://reactrouter.com/en/main/components/scroll-restoration) from react-router.